toolchain/mingw-toolchain.xml

```diff
@@ -60,6 +65,8 @@
 
 <copyFile toolId="exe" name="libgcc_s_dw2-1.dll" from="${MINGW_ROOT}/bin" allowMissing="true" unless="no_shared_libs"/>
 <copyFile toolId="exe" name="libstdc++-6.dll" from="${MINGW_ROOT}/bin" allowMissing="true" unless="no_shared_libs"/>
+<copyFile toolId="exe" name="libwinpthread-1.dll" from="${MINGW_ROOT}/sys-root/mingw/bin" allowMissing="true" unless="no_shared_libs"/>

```

Comment:
the compiled Win32 application looks for libwinpthread-1.dll, including this after compile will resolve this issue.
By far, there were two possible location of libwinpthread-1.dll so two copyFile was added.